### PR TITLE
pc_vga.cpp: CRTC start address is supposed to be shifted left by 2, from 86Box [Melissa Goad]

### LINKED ISSUE #2954: One Must Fall 2097 in ct486 latches the VGA start address at the wrong time
Problematic code is in src/devices/video/pc_vga.cpp, and m_vblank_timer seems to have the wrong time given to it.

The VGA start address latches a value of 0xA4, when it seemingly should latch 0.

### PR DESCRIPTION
**Pull request by MoochMcGee**
Fixes #2954

### CODE REVIEW

File: src/devices/video/pc_vga.cpp [hunk 1]
```diff
@@ -1489,6 +1489,7 @@ void vga_device::crtc_reg_write(uint8_t index, uint8_t data)
 			break;
 		case 0x0c:
 		case 0x0d:
+			//if(machine().first_screen()->vpos() < (vga.crtc.vert_retrace_start)) break;
```
Comment (DopefishJustin) [inline src/devices/video/pc_vga.cpp]: Please remove stuff like this before committing.

**Comment by DopefishJustin** [comment]
Confirmed this does fix the issue with OMF 2097 but it causes flickering in DOOM.

**Comment by MoochMcGee** [comment]
@DopefishJustin Investigating a fix for DOOM that isn't a hack.

**Comment by MoochMcGee** [comment]
@DopefishJustin this was the only fix i could come up with, unfortunately.

**Comment by MoochMcGee** [comment]
Wait, this breaks the bleed out effect, investigating further.

**Comment by MoochMcGee** [comment]
@DopefishJustin OMF and DOOM now both work.

**Comment by DopefishJustin** [comment]
Looks good from a functional point of view now, the split screen glitch in OMF 2097 and a similar one in Kyrandia are fixed, and no regressions noted so far. I can't speak to whether it makes sense from a hardware perspective.

**Pull request merged**